<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>407</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Remote UAV 1.15 03/06/2021</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_TCP">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>TCP</string>
    </property>
    <widget class="QPushButton" name="connectToHost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Reconnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>try reconnect</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_cargo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Cargo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="connectCargo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Link</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SetCargoPeriod">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>65</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set period</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SetCargoDuration">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>65</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set duration</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SetCargoFrequency">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>65</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set frequency</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_cargoPeriod">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>105</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_cargoDuration">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>105</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>11</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_CargoFrequency">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>105</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>6800</number>
     </property>
     <property name="maximum">
      <number>13600</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CargoGeneratorOFF">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>25</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CargoGeneratorON">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>25</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_ctrl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>5</y>
      <width>171</width>
      <height>396</height>
     </rect>
    </property>
    <property name="title">
     <string>Single point</string>
    </property>
    <widget class="QPushButton" name="sendCtrlMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>355</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveToPointRequestLongitude">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_pointInCoordinates">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>255</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>PointInCoordinates</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="pointInCoordinatesLatitude">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>29</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>60</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>latitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_56">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>longitude</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pointInCoordinatesLongitude">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>59</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>53</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>longitude</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveToPointRequestLatitude">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>25</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>60</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>78</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>altitude</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_servo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Servo</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="servoAngle">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_54">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>targetAngle</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>actionTime</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="holdPointActionTime">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>28</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>latitude</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveToPointRequestAltitude">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>75</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_course">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>125</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Course</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="moveToPointRequestCourse">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>course</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="log">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>250</y>
      <width>494</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clearLog">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>✖</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>5</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Track</string>
    </property>
    <widget class="QPushButton" name="sendTrack">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>25</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send track</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="loadTrack">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load CSV</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="expand">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>25</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Single &gt;&gt;</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="previewTrack">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>25</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_retranslator">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>100</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Retranslator</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="retranslator_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.112</string>
     </property>
    </widget>
    <widget class="QPushButton" name="retranslator_enable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="retranslator_port">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="value">
      <number>32179</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Reconnect_retranslator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>115</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>try reconnect</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>host</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="naviLog">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>280</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>🌏</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
